CONTRIBUTING.md

```
@@ -0,0 +1,8 @@
+# Contributing to TravelSplit
+
+Stuff To Do:
+https://linear.app/travelsplit/team/TRA/backlog
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)